var/spack/repos/builtin/packages/vtk-m/package.py

```diff
@@ -81,13 +71,14 @@ class VtkM(CMakePackage, CudaPackage):
     depends_on("tbb", when="+tbb")
     depends_on("mpi", when="+mpi")
 
-    for amdgpu_value in amdgpu_targets:
+    # Propagate AMD GPU target to kokkos for +rocm
+    for amdgpu_value in ROCmPackage.amdgpu_targets:
```

Comment:
Note I removed the amdgpu_targets for this because kokkos is already selecting the correct GPU targets. The new concretizer handles this correctly, so I removed it so when kokkos is able to handle other AMD GPU targets VTKm doesn't lag behind.